supervillains.json

```diff
@@ -272,6 +273,7 @@
 	"Hela",
 	"Hellgrammite",
 	"High Evolutionary",
+	"Him",
```

Comment:
This is too generic. Can you drop this one?

Reply:
You're absolutely right. That wasn't very thoughtful of me either. Dropped it.